### PR TITLE
ntlite: Add version 2024.7.9999

### LINKED ISSUE #13386: [Request]: NTLite
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

NTLite

### Description

Local control for updating and editing Windows images and deployments. For IT professionals and enthusiasts

### Homepage

https://www.ntlite.com/

### Download Link(s)

https://downloads.ntlite.com/files/NTLite_setup_x64.exe https://downloads.ntlite.com/files/NTLite_setup_x86.exe https://downloads.ntlite.com/files/NTLite_beta_setup_x64.exe

### Some Indication of Popularity/Repute

The tool has been reviewed by trusted tech channels, listed at https://www.ntlite.com/support/ and has a dedicated community forum at https://www.ntlite.com/community/index.php for any issues.

Additional info: 
- the install tool provides a portable mode, so it should be possible to wrap it via Scoop.
- a manifest can be found at [this](https://github.com/okibcn/ScoopMaster/blob/master/bucket/NTLite-Portable.json) custom bucket, in case it could help.
- while the tool offers a paid license, the free version is more than enough to do all the common modifications you'd see in guides and tutorials for it.

### PR DESCRIPTION
**Pull request by bt4ibwem8**
Closes #13386

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ntlite.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                }
+            },
+            "32bit": {
+                "url": "https://downloads.ntlite.com/files/NTLite_setup_x86.exe",
+                "hash": {
+                    "url": "https://www.ntlite.com/download/",
+                    "regex": "NTLite_setup_x86.exe.*SHA-256: $sha256"
+                }
+            }
+        }
```
Comment (aliesbelik) [inline bucket/ntlite.json:60]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://downloads.ntlite.com/files/NTLite_setup_x64.exe"
            },
            "32bit": {
                "url": "https://downloads.ntlite.com/files/NTLite_setup_x86.exe"
            }
        },
        "hash": {
            "url": "https://www.ntlite.com/download/",
            "regex": "$basename.*?SHA-256: $sha256"
        }
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Review by bt4ibwem8 — COMMENTED** [review]
thanks

**Comment by bt4ibwem8** [comment]
@aliesbelik
Did i approved your request correctly? It still shows "Changes requested" in red color so i am not sure if i need to do something or not...

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**